primitives/core/src/crypto.rs

```diff
@@ -576,6 +576,10 @@ ss58_address_format!(
 		(46, "reserved46", "Reserved for future use (46).")
 	Reserved47 =>
 		(47, "reserved47", "Reserved for future use (47).")
+	SocialTestAccount =>
+		(251, "test-social-network", "Test Social Network, standard account (*25519).")
```

Comment:
For test nets you can just use the generic substrate prefix.

Reply:
Sure no problem, I removed it.